### ISSUE TITLE
The loss does not converge during training

### ISSUE THREAD

**Issue by celiazzx** [daooshee/HLA-Face-Code#13]
During the pre-training of the classification head and the comparison learning head, as well as during the Joint High-Low Adaptation training, the loss does not converge and oscillates repeatedly. Can you provide me with your gradient descent curve?
